contracts/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "contracts",
+  "version": "1.0.0",
+  "repository": "https://SeiyaKobayashi@github.com/SeiyaKobayashi/traininig-optimization-on-ml-model-with-zkp.git",
+  "author": "SeiyaKobayashi <seiya.kobayashi.work@gmail.com>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "compile": "yarn hardhat compile",
```

Comment:
I think `yarn` is not needed here because the bin files are installed as dependencies under `node_modules/.bin`.
Do you have any intention on using `yarn hardhat compile` instead of just `hardhat compile`?

Reply:
Fixed (891dfd9195eae857bdfcbf63dee2d3d6f4fd616c).